Dockerfile

```diff
@@ -1,7 +1,9 @@
+ARG REQFILE=requirements-prod.txt
+
 FROM python:3.6-slim
 MAINTAINER "EEA: IDM2 C-TEAM" <eea-edw-c-team-alerts@googlegroups.com>
 
-ARG REQFILE=requirements-prod.txt
+ARG REQFILE
```

Comment:
Declared at the top of the file and here too?

Reply:
Yes, this argument wasn't working at all. I investigated why and it turns out this is the way to fix it. Now we don't have to install the dev requirements every time we attach to the container :)

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact